apis/contour/v1beta1/tlscertificatedelegation.go

```diff
@@ -40,7 +40,7 @@ type CertificateDelegation struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// TLSCertificateDelefgation is an TLS Certificate Delegation CRD specificiation.
+// TLSCertificateDelefgation is an TLS CACertificate Delegation CRD specificiation.
```

Comment:
Please revert this, I think it was an unintended change.

Reply:
The comment also has a typo: TLSCertificateDele**f**gation